book/frontend/tutorial/step_8.md

```diff
@@ -0,0 +1,58 @@
+# Step 8
+
+You reached the last step of the tutorial!
+
+In our projects we almost always use the (FlexView)[https://github.com/buildo/react-flexview] component to compose our layout and to wrap our components, instead of the classic `<div>`. Let's try to do the same here!
```

Comment:
I can link to the intro of the book of flexview, that shows an example of why it is better!